docs/source/models/concept_models/vivarium_mncnh_portfolio/maternal_disorders_module/module_document.rst

```diff
@@ -68,7 +68,7 @@ There are specific variables that will affect these cause models, summarized in
     - Instructions
     - Note
   * - Azithromycin coverage
-    - :ref:`Intrapartum interventions <2024_vivarium_mncnh_portfolio_intrapartum_interventions_module>`
+    - :ref:`Azithromycin intervention page <azithromycin_intervention>`
     - Maternal sepsis incidence rate
     - Relative risk: 0.65 (95% CI: 0.55, 0.77)
     - `Tita et al 2023 <https://www.ajog.org/article/S0002-9378(22)02210-4/fulltext#undfig1>`_ 
```

Comment:
```suggestion
    - :ref:`Intrapartum interventions <2024_vivarium_mncnh_portfolio_intrapartum_interventions_module>`
    - Maternal sepsis incidence rate
    - See the :ref:`Azithromycin intervention page <azithromycin_intervention>`
    - 
```

We still want to link to the intrapartum intervention module (how we are organizing the concept model), but instructions for how to apply effect will link to the intervention model document. We also want to be sure to delete this outdated information on effect sizes!